build.gradle

```diff
@@ -239,6 +239,14 @@ docker {
     baseImage = 'eclipse-temurin:17-jre-jammy'
     maintainer = 'Simon Templer "simon@wetransform.to"'
     images = ["wetransform/${project.name}:${project.version}", "wetransform/${project.name}:latest"]
+    // apply JAVA_OPTS to docker plugin as the default args configured in build does not apply to docker plugin
+    jvmArgs = ['-Dcache.level1.enabled=false',
```

Comment:
Since the same args are also set in another place in the same file I would prefer to have a single place to maintain them.
For example some place before define a variable that is then reused:
```
def defaultJvmArgs = [...]

...
  
  jvmArgs = defaultJvmArgs
```

Reply:
Yes, makes sense. I have modified the code.